crates/agglayer-node/src/node.rs

```diff
@@ -82,25 +88,48 @@ impl Node {
             }
         };
 
+        let aggregator_task = AggregatorNotifier::new();
+        let clock_subscription =
+            tokio_stream::wrappers::BroadcastStream::new(clock_ref.subscribe()?)
+                .filter_map(|value| value.ok());
+
+        let (_data_sender, data_receiver) = mpsc::channel(
+            config
+                .certificate_orchestrator
+                .input_backpressure_buffer_size,
+        );
+
+        let proof_orchestrator_handle = CertificateOrchestrator::builder()
```

Comment:
```suggestion
        let certificate_orchestrator_handle = CertificateOrchestrator::builder()
```

I see other occurrences of `proof_orchestrator` too